assets/sass/_mixins.scss

```diff
@@ -101,6 +101,15 @@
   }
 }
 
+@mixin set-column-width($column-number) {
+  $one-column-size: 8.333333333333333%;
+  $calculated-number-of-columns-size: $column-number * $one-column-size;
+  -webkit-box-flex: 0;
+  -ms-flex: 0 0 #{$calculated-number-of-columns-size};
+  flex: 0 0 #{$calculated-number-of-columns-size};
+  max-width: #{$calculated-number-of-columns-size};
```

Comment:
Please can we use grid for width's on the Hero Banner rather than percentages. See ```.content-container-grid``` class as an example.

Reply:
The mixin is removed, there is a new logic inside hero-banner.scss similer to .content-container-grid class